src/details_window.cr

```diff
@@ -50,13 +50,13 @@ class DetailsWindow < Window
       @card.activities.map { |activity| CardAction.new(activity) }.each do |activity|
         pad.attron(NCurses::Attribute::BOLD | NCurses::Attribute::UNDERLINE)
         wrap(activity.title, @width - 2).each_line do |line|
-          pad.addstr(line)
-          pad.addstr("\n") unless line.ends_with?("\n")
+          pad.addstr(line.rstrip)
+          pad.addstr("\n") unless line.size == @width - 2
         end
         pad.attroff(NCurses::Attribute::BOLD | NCurses::Attribute::UNDERLINE)
         wrap(activity.description, @width - 2).each_line.with_index do |line, i|
-          pad.addstr(line)
-          pad.addstr("\n") unless line.ends_with?("\n")
+          pad.addstr(line.rstrip)
+          pad.addstr("\n") unless line.size == @width - 2
```

Comment:
Should we be checking for `line.rstrip.size == @width - 2`

Reply:
No, because there is no newline. I guess `rstrip` is unnecessary.

```
➜ crystal eval 'p "foo\nbar\nbaz".each_line.to_a'
["foo", "bar", "baz"]

➜ ruby -e 'p "foo\nbar\nbaz".each_line.to_a'     
["foo\n", "bar\n", "baz"]
```

Reply:
Ah, 👍